test/performance/collectd/parameters.yml

```diff
@@ -1,12 +1,12 @@
-receivers: "collectd"
+receivers: ["collectd"]
 
 test_case: "collectd_performance"
 validate_type: "performance"
 data_type: "metrics"
 # Number of metrics to be sent or number of log lines being written  each minute
 values_per_minute: "<values_per_minute>"
-# Number of seconds the agent should run and collect the metrics. In this case, 5 minutes
-agent_collection_period: 300 
+# Number of seconds the agent should run and collect the metrics. In this case, 3 minutes
+agent_collection_period: 180 
```

Comment:
Might need more than 3 minutes to get an accurately measure steady state performance of CWA. Why do you feel confident to reduce it from 5?

Reply:
First minute and second minute would be enough for the buffer to fill our metrics/traces/logs. However, I agree with you `accurately measure steady state performance of CWA` the more time it runs since the average for 5 minutes would mostly similar to 7 minutes within an error rate of 10%.

Reply:
Yes agree. Changing to the original 5 minutes instead of 3 minutes. It should be fine for waiting 5 minutes for increasing accuracy.